Changes

```diff
@@ -330,6 +330,9 @@ Working version
   on Power and Z System
   (Xavier Leroy, review by Nicolás Ojeda Bär)
 
+- #9440: for a type extension constructor with parameterised arguments,
+  REPL displayed <poly> for each as opposed to the concrete values used.
+
```

Comment:
Could you include a credit line with your preferred name (we have a preference for full names if you are fine with it), and potentially reviewer names?